docs/admin/install/serving/ingress-implementations.md

```diff
@@ -0,0 +1,11 @@
+# Unofficial Ingress Implementations
+
+In addition to the Knative Ingress implementations in the ([install a networking layer](../install-serving-with-yaml/#install-a-networking-layer)) section,
+there are other Knative Ingress implementations.
+
+Here is the list of the ingress implementations (alphabetically):
+
+- [Emissary-Ingress (Ambassador API Gateway)](https://www.getambassador.io/docs/edge-stack/latest/howtos/knative/): Emissary-Ingress is an open-source Kubernetes-native API Gateway + Layer 7 load balancer + Kubernetes Ingress built on Envoy Proxy.
+- [Kong](https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/using-kong-with-knative/): Kong is an open source API Gateway built for hybrid and multi-cloud environments.
+
+These implementations are not officially supported by Knative community at this moment, but the implementation will be added to the main guide once it is ready for general use.
```

Comment:
I'm not sure I would add this to main docs at all yet if it's not supported tbh; they should probably be documented for use with Knative in some other repo or on in the Kong docs, etc.